.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: Release
+
+on:
+  push:
+    # tags:
+    #   - v*
+
+permissions:
+  contents: write
+
+jobs:
+  create_draft_release:
+    runs-on: ubuntu-22.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Create draft release
```

Comment:
I think we can create the release if it doesn't exist yet, no need for draft. :)